src/Compilers/Core/Portable/Emit/OperandType.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Emit
+{
+
+    using System;
+#pragma warning disable RS0016 // Add public types and members to the declared API
+
+    [System.Runtime.InteropServices.ComVisible(true)]
+    public enum OperandType
```

Comment:
I had a look, I think we could get away with duplicating `FILETYPE` _and_ `OperandType` and just marking them internal. It doesn't appear we expose them anywhere. 

@tmat Thoughts?